packages/desktop-client/src/components/reports/Header.tsx

```diff
@@ -66,82 +67,68 @@ export function Header({
     <View
       style={{
         padding: 20,
-        paddingTop: 0,
         flexShrink: 0,
       }}
     >
-      <View
+      <SpaceBetween
```

Comment:
Various small re-alignments in cashflow & networth pages on small devices (i.e. mobile). To make the filters look slightly better on small resolutions.